src/python/CRABClient/Commands/status.py

```diff
@@ -569,14 +569,18 @@ def terminate(states, jobStatus, target='no output'):
         # And if the dictionary is not empty, print it
         for jobtype, currStates in toPrint:
             if currStates:
-                automaticSplittFAQ = 'https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3FAQ#What_is_the_Automatic_splitting'
-                if automaticSplitt and (not self.options.long  or  jobtype == 'Probe jobs'):
+                if automaticSplitt and jobtype == 'Probe jobs':
+                    automaticSplittFAQ = 'https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3FAQ#What_is_the_Automatic_splitting'
                     self.logger.info("\nThe jobs splitting of this task is 'Automatic', please refer to this FAQ for a description of the jobs status summary:\n%s", automaticSplittFAQ)
                 total = sum(currStates[st] for st in currStates)
                 state_list = sorted(currStates)
                 self.logger.info("\n{0:32}{1} \t\t {2}".format(jobtype + ' status:', self._printState(state_list[0], 13), self._percentageString(state_list[0], currStates[state_list[0]], total)))
                 for jobStatus in state_list[1:]:
                     self.logger.info("\t\t\t\t{0} {1}".format(self._printState(jobStatus, 13), self._percentageString(jobStatus, currStates[jobStatus], total)))
+                if jobtype == 'Probe jobs':
+                    scheddName = self.schedd.replace("crab3@","")
+                    scheddNum = self.schedd.replace("crab3@vocms","").replace(".cern.ch","")
+                    self.logger.info("Probe stage log:\t\t%s", self.webdir.replace("http://","https://"+self.serverurl).replace(scheddName+"/mon","/scheddmon/"+scheddNum)+"/AutomaticSplitting_Log0.txt")
```

Comment:
should be able to proxiedwebdir here, insread of this complicated URL editing

Reply:
OK thanks!